### DISCUSSION TITLE
Published in registry but wallet app not appearing on iOS list

### DISCUSSION THREAD

**Discussion by catboop** [WalletConnect/walletconnect-monorepo#1491] [Developer Support]
What are the possible reasons our wallet app would not be appearing in a custom list of wallet options in the QR code modal on iOS?

**Comment by looter** [comment]
This is likely because the Mobile Linking section is blank for your explorer app entry.

**Reply by catboop** [reply]
Hmm, what if we have a deep link and universal link for Mobile Linking section? Is it possible we are listing our wallet name incorrectly in qrcodeModalOptions? Is it case sensitive?
